components/WhiteLine.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+interface PropsInterface {
+    className?: string
+}
+
+export default function WhiteLine(props: PropsInterface) {
+    const { className } = props;
+
+    return (
+        <div className={`h-2 bg-white ${className}`} />
```

Comment:
hmmm not sure if we really need nga usa ka component for a white line, can't we just put this in the page directly? unsa man and pros ani? like if gamiton nato on other components kay one place rata mag change? pero how many times do we really use this diay? basin too much abstraction raba nya mu pile up raba na ang file size if daghan ta components, hmmm

Reply:
also dili ba `hr` imo gina find? hehehehehehe